kpi/tests/api/test_api_environment.py

```diff
@@ -33,41 +35,44 @@ def setUp(self):
             'community_url': constance.config.COMMUNITY_URL,
             'frontend_min_retry_time': constance.config.FRONTEND_MIN_RETRY_TIME,
             'frontend_max_retry_time': constance.config.FRONTEND_MAX_RETRY_TIME,
-            'project_metadata_fields': lambda x: \
-                self.assertEqual(len(x), len(json.loads(constance.config.PROJECT_METADATA_FIELDS))) \
-                and self.assertIn({'name': 'organization', 'required': False}, x),
-            'user_metadata_fields': lambda x: \
-                self.assertEqual(
-                    len(x), len(json.loads(constance.config.USER_METADATA_FIELDS))
-                ) and self.assertIn({'name': 'sector', 'required': False}, x),
-            'sector_choices': lambda x: \
-                self.assertGreater(len(x), 10) and self.assertIn(
-                    ("Humanitarian - Sanitation, Water & Hygiene",
-                     "Humanitarian - Sanitation, Water & Hygiene"),
-                    x
+            'project_metadata_fields': lambda x: self.assertEqual(
+                len(x),
+                len(json.loads(constance.config.PROJECT_METADATA_FIELDS)),
+            ) and self.assertIn({'name': 'organization', 'required': False}, x),
+            'user_metadata_fields': lambda x: self.assertEqual(
+                len(x), len(json.loads(constance.config.USER_METADATA_FIELDS))
+            ) and self.assertIn({'name': 'sector', 'required': False}, x),
+            'sector_choices': lambda x: self.assertGreater(
+                len(x), 10
+            ) and self.assertIn(
+                (
+                    "Humanitarian - Sanitation, Water & Hygiene",
+                    "Humanitarian - Sanitation, Water & Hygiene",
                 ),
+                x,
+            ),
             'operational_purpose_choices': (('', ''),),
-            'country_choices': lambda x: \
-                self.assertGreater(len(x), 200) and self.assertIn(
-                    ('KEN', 'Kenya'), x
-                ),
-            'interface_languages': lambda x: \
-                self.assertEqual(len(x), len(settings.LANGUAGES)),
+            'country_choices': lambda x: self.assertGreater(
+                len(x), 200
+            ) and self.assertIn(('KEN', 'Kenya'), x),
+            'interface_languages': lambda x: self.assertEqual(
+                len(x), len(settings.LANGUAGES)
+            ),
             'submission_placeholder': SUBMISSION_PLACEHOLDER,
             'asr_mt_features_enabled': False,
             'mfa_enabled': constance.config.MFA_ENABLED,
-            'mfa_localized_help_text': lambda i18n_texts: {
-                lang: markdown(text)
-                for lang, text in json.loads(
-                    constance.config.MFA_LOCALIZED_HELP_TEXT.replace(
-                        '##support email##',
-                        constance.config.SUPPORT_EMAIL
-                    )
-                ).items()
-            },
```

Comment:
Oops, it was impossible for this to fail. Always make sure your tests can fail! I'll take some blame here, too, because making values automatically get `assertEqual()`ed while requiring callables to do their own assertions leaves some clarity to be desired.